### ISSUE TITLE
Tracy being enabled causes some websites to incorrectly redirect

### ISSUE THREAD

**Issue by MichaelBlakeOIT** [nccgroup/tracy#118]
When going through shopify at `https://anystorehere.myshopify.com`, I get redirected to `https://anystorehere.myshopify.comhttps://anystorehere.myshopify.com`. This does not happen when Tracy is disabled. I have tried this in both Chrome and Firefox.

**Comment by jacobheath** [comment]
Hmm, do you have an example shopify store I can try?

**Comment by MichaelBlakeOIT** [comment]
@jacobheath I can't find any examples that don't require being logged in, but if you're logged into a shop, and visit `https://whateveryourshopnameis.myshopify.com/admin/products` you will be redirected to `https://whateveryourshopnameis.myshopify.com/admin/productshttps://whateveryourshopnameis.myshopify.com/admin/products`

I'll continue looking around for an area that doesn't require logging in or another site that has this behavior.

**Comment by jacobheath** [comment]
I created a free trial and was able to reproduce the issue. Diving into it now.

**Comment by jacobheath** [comment]
I think I figured it out. I was keeping when XHRs were being made in the current `this` context. Turns out, I wasn't using a unique enough variable name because the code on Shopify was also using that same variable name to set the `window.location`. Luckily, I don't need this state anymore. I am pushing a commit that removes the code. In a few minutes, update your version of Tracy and give it a shot and let me know if it works.
